developer_manual/app/info.rst

```diff
@@ -117,6 +117,13 @@ ownCloud allows to specify four kind of "types". Currently supported "types":
 
 * **logging**: apps which implement a logging system
 
+* **prevent_group_restriction**: apps which can not be enabled for specific groups (e.g. notifications app).
```

Comment:
I don;'t understand what this means?
Does it mean:
"apps which can be disabled for specific groups"?
i.e. the admin can specify group(s) that do not have the app. The app would then be available to all other users, apart from those in the specified group(s).